elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -52,7 +52,9 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, $backwards-compatibility);
   @include pfe-c-typography($type: text, $sizing: md, $base: true, $region: tab, $light-dom-heading: true);
   
   #tab {
-    display: inline-block;
+    display: inline-flex;
```

Comment:
This is solid btw.  I put really janky content in here to try and break it and it stays nicely centered. :)